tests/handlers/test_sync.py

```diff
@@ -1062,14 +1098,16 @@ def generate_sync_config(
     user_id: str,
     device_id: Optional[str] = "device_id",
     filter_collection: Optional[FilterCollection] = None,
+    use_state_after: bool = False,
 ) -> SyncConfig:
     """Generate a sync config (with a unique request key).
 
     Args:
         user_id: user who is syncing.
         device_id: device that is syncing. Defaults to "device_id".
         filter_collection: filter to apply. Defaults to the default filter (ie,
-           return everything, with a default limit)
+        return everything, with a default limit)
```

Comment:
This line appears to have been accidentally dedented.

```suggestion
            return everything, with a default limit)
```